architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,23 @@ endif::openshift-dedicated,openshift-online[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+[[architecture-projects-openshift-default-project]]
+===The `openshift` and `default` projects
+
+{product-title} comes with two projects by out of the box:
+
+*`openshift`*
+The `openshift` project can be accessed by users of any projects. This is mainly
+used to house objects for day-to-day tasks. These include any application
+objects for access by multiple projects, such as templates and images. These
+objects should be those that do not require communication between the pods.
+
+*`default`*
```

Comment:
We want "default" to die.  The advice here is rather old.

Reply:
@deads2k So you're recommending that i take out the stuff about the default project?